frontend/packages/noobaa-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -68,7 +69,11 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
     promiseObj
       .then((obj) => {
         dispatch({ type: 'setIsLoading', value: false });
-        history.push(resourceObjPath(obj, referenceFor(obj)));
+        history.push(
+          `/k8s/ns/${ns}/clusterserviceversions/${
+            clusterServiceVersion.metadata.name
```

Comment:
use getName function from console shared.
```suggestion
            clusterServiceVersion.metadata.name
```